tasks/eva_3156/eva-biosamples.json

```diff
@@ -0,0 +1,175 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Not sure if it can be used but noticed that there seems to be a [newer draft](https://json-schema.org/draft/2020-12/release-notes.html)....